picky/src/x509/pkcs7.rs

```diff
@@ -0,0 +1,76 @@
+use crate::hash::UnsupportedHashAlgorithmError;
+use crate::pem::Pem;
+use crate::x509::certificate::CertError;
+use crate::x509::utils::{from_der, from_pem, from_pem_str, to_der, to_pem};
+use picky_asn1::restricted_string::CharSetError;
+use picky_asn1_der::Asn1DerError;
+use picky_asn1_x509::algorithm_identifier::UnsupportedAlgorithmError;
+use picky_asn1_x509::pkcs7::Pkcs7Certificate;
+
+use thiserror::Error;
+
+type Pkcs7Result<T> = Result<T, Pkcs7Error>;
+
+pub const AUTHENTICODE_ATTRIBUTES_COUNT: usize = 3;
+
```

Comment:
We probably don't need that at all? Is that something you use in consumer code?

```suggestion
```

Reply:
Yes, We don't need it. I forget to remove it.